app/controllers/bikes_controller.rb

```diff
@@ -266,14 +266,14 @@ def theft_templates
   # are used as haml header tag text in the corresponding templates.
   def bike_templates
     {}.with_indifferent_access.tap do |h|
-      h[:bike_details] = translation(:bike_details, scope: %i[controllers bikes edit])
-      h[:photos] = translation(:photos, scope: %i[controllers bikes edit])
-      h[:drivetrain] = translation(:drivetrain, scope: %i[controllers bikes edit])
-      h[:accessories] = translation(:accessories, scope: %i[controllers bikes edit])
-      h[:ownership] = translation(:ownership, scope: %i[controllers bikes edit])
-      h[:groups] = translation(:groups, scope: %i[controllers bikes edit])
-      h[:remove] = translation(:remove, scope: %i[controllers bikes edit])
-      h[:report_stolen] = translation(:report_stolen, scope: %i[controllers bikes edit]) unless @bike.stolen?
+      h[:bike_details] = translation(:bike_details, controller_method: :edit)
```

Comment:
Does this really change? This is the template name, is it not? Shouldn't this be translated somewhere else?

Reply:
> Does this really change?

taking `this` to mean the change in keyword arg used from `scope:` to `controller_method:`:

It doesn't necessarily need to change, but I think it's preferable to prefer the `controller_method` kwarg where possible — doing so is DRYer and has smaller scope for error.

> This is the template name, is it not?

taking `this` to mean `h[:bike_details] = <value>`:

The hash key is the template name and the value is a title string rendered in the template. The hash overall provides the navigation menu.

> Shouldn't this be translated somewhere else?

taking `this` to mean all the translations in `bike_templates` and `theft_templates`:

These hashes couple filenames to query params to rendered strings to the navigation menu, so I'd rather they weren't defined at all. But, conditional on this abstraction, translating these strings at their origin point seems simpler than doing so everywhere they're used in templates. I'm open to refactoring it though — concretely, we could map from template names to translation keys instead of mapping to strings.

Reply:
I was being stupid. It all makes sense now!